src/Hintman/Core/Delta.hs

```diff
@@ -0,0 +1,32 @@
+{- | This module contains subset of all possible diff deltas that can
+be handled by Hintman. We consider for hints only the following deltas:
```

Comment:
Nit: "We consider for hints only the following deltas" sounds a bit awkward.
I'd suggest something like "Hintman processes the deltas that satisfy the following criteria".